playbooks/container-runtime/config.yml

```diff
@@ -2,5 +2,7 @@
 - import_playbook: ../init/main.yml
   vars:
     skip_verison: True
+    l_openshift_version_set_hosts: "all:!all"
```

Comment:
Not sure why is it needed - that would include hosts from `all` group - and all hosts from any other group?

Reply:
This resolves to no hosts.  Basically this is the only way (or we could use a group other than 'all') to specifically say we don't want to run tasks on any hosts.  In this case, we're just avoiding skipped tasks.

Reply:
I think I'd be in favor of creating a group in ../init/evaluate_groups.yml that creates an empty group.  That way we don't maintain all of these `all:!all` snippets in every place.  We already require init to be run and our inventory groups to be defined.

Thoughts?

Reply:
+1 to a separate group for that, but that shouldn't block this PR.

I've also noticed some parts use `skip_version` and some `skip_verison`, so I'd prefer to create improvement PRs after this one is merged

Reply:
I'm going to fix the skip_verison typo.

I'm not a fan of evaluate_groups, I don't want to add to it, but it might be clearer to end-users what we're doing if we have an oo_empty_group or something.